### ISSUE TITLE
Transmission-daemon instance associated with WhatManager memory leak after upgrading to Stretch

### ISSUE THREAD

**Issue by desultory** [karamanolev/WhatManager2#89]
When I upgraded to stretch, the transmission instance used by WhatManager keeps using up more RAM until it is all used.  The transmission instance I have running that is not associated with WhatManager does not behave like this.

**Comment by MADindustries** [comment]
A couple questions:

1. Did you build transmission-daemon from source when you installed WM (using the included script) or did you use a package manager?

2. Is your other transmission instance a transmission-daemon instance (as opposed to gtk/qt) from the same source or is it from a different source?

3. Are both transmission instances the same version?

**Comment by desultory** [comment]
both transmission instances are the same version, I believe i used the package manger version and they are both transmission daemons, both being 2.92.  This memory leak didn't happen until I upgraded to stretch.  I have the cache size set lower than default but that didn't fix the issue.  This transmission service eats about 1gb of ram an hour until everything gets pushed to the swap.

**Comment by desultory** [comment]
http://i.imgur.com/tfuLYKq.png

Here is what the usage looks like in ESXi 6.5u1.  I ran a fsck a few days ago and it found errors but it looks like that didn't fix the memory leak.  I'm considering scrapping this VM and making a new one.  I already moved every service other than transmission off of this server and onto it's own server because before I was running 2 transmission daemons, 1 apache server, 1 teamspeak server, 1 unrealircd server, a kiwi gateway, and a bunch of other nonsense on this one vm.
